awesome_dashboard/static/src/dashboard/dashboard_item/pie_chart/pie_chart_card.js

```diff
@@ -0,0 +1,22 @@
+/** @odoo-module */
+
+import { Component,xml } from "@odoo/owl";
+import { PieChart } from "./pie_chart";
+
+export class PieChartCard extends Component {
+    static template = xml`
+    <t t-name="awesome_dashboard.PieChartCard" owl="1">
+        <t t-esc="props.title"/>
+        <PieChart data="props.values" label="''"/>
+    </t>`;
```

Comment:
```suggestion
    static template = "awesome_dashboard.PieChartCard";
```
a cleaner way would be to put the xml inside a xml file